x/emissions/module/rewards/rewards_test.go

```diff
@@ -2861,6 +2861,25 @@ func (s *RewardsTestSuite) TestRewardForTopicGoesUpWhenRelativeStakeGoesUp() {
 	err = s.emissionsAppModule.EndBlock(s.ctx)
 	require.NoError(err)
 
+	worker1InclusionNum, err := s.emissionsKeeper.GetCountInfererInclusionsInTopic(s.ctx, topicId0, s.addrsStr[workerIndexes[0]])
+	require.NoError(err)
+	require.Positive(worker1InclusionNum)
```

Comment:
can we do better than just checking it's positive, this seems very lazy

we should know what we expect the number to be during this (and every other rewards) test